packages/swapper/src/swappers/ArbitrumBridgeSwapper/utils/fetchArbitrumBridgeSwap.ts

```diff
@@ -5,13 +5,13 @@ import type {
 } from '@arbitrum/sdk/dist/lib/dataEntities/transactionRequest'
 import type { ChainId } from '@shapeshiftoss/caip'
 import { ethAssetId, ethChainId, fromAssetId } from '@shapeshiftoss/caip'
+import type { EvmChainAdapter, EvmChainId } from '@shapeshiftoss/chain-adapters'
 import { type Asset, KnownChainIds } from '@shapeshiftoss/types'
+import { assertUnreachable, bn } from '@shapeshiftoss/utils'
+import { getFees } from '@shapeshiftoss/utils/dist/evm'
```

Comment:
question: Do we really need to import this `dist` folder? I would expect to be able to import the source file, it makes it 100% needed to build this as a dependency in order to consume it, making the DX a bit harder

Reply:
Yes for now, see https://github.com/shapeshift/web/pull/7249#discussion_r1663454578 where we should be moving things into more ergonomic locations to avoid importing from dist